captura/views.py

```diff
@@ -1,3 +1,59 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from django.contrib.auth.decorators import user_passes_test, login_required
+from perfiles_usuario.utils import is_capturista
 
-# Create your views here.
+
+@login_required
+@user_passes_test(is_capturista)
+def pending_studies(request):
+    """ View to see the list of pending studies.
+    TODO. This is a dummy view, therefore it may
+    be changed in the future.
+
+    """
+
+    return render(request, 'administracion/dashboard_users.html')
```

Comment:
Same as above, it is the template that sends each type of user to its dashborad. In this case the list of pending studies for a Capturista is its dashboard